### ISSUE TITLE
Can select buttons along the top, but they do not do anything.

### ISSUE THREAD

**Issue by Shubeans** [7yl4r/the-oregon-trajectory#119]
Can select 'distance travelled' 'rations' 'fuel' 'distance to moon', sound and animation plays but nothing happens.
One would assume that these take the user to page or create a popup. If not they should not play sfx or animations.

Tested on:
Chrome version 43.0

**Comment by 7yl4r** [comment]
You're correct, these buttons have unimplemented functions. They are meant to provide additional detail and the ability to alter pace, rations, etc.
